### ISSUE TITLE
AttributeError: module 'bbprep.generators' has no attribute 'TargetTorsion'

### ISSUE THREAD

**Issue by Joe-Anslow** [andrewtarzia/bbprepared#12]
When running tutorial_code.py, this error occurs:
```
Traceback (most recent call last):
  File "/home/cmja2/STK_Calculations/BBPrepared-Testing/tutorial_code.py", line 161, in <module>
    main()
  File "/home/cmja2/STK_Calculations/BBPrepared-Testing/tutorial_code.py", line 154, in main
    planarfy()
  File "/home/cmja2/STK_Calculations/BBPrepared-Testing/tutorial_code.py", line 66, in planarfy
    target_torsions=bbprep.generators.TargetTorsion(
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'bbprep.generators' has no attribute 'TargetTorsion'
```

**Comment by andrewtarzia** [comment]
I assumed you are talking about this code: https://gist.github.com/andrewtarzia/23e7843365bad76277322e207d95e627

I tested it with Python 3.11 and indeed there was a change in the interface that is not reflected in the gist. I have updated the gist.

Mainly, you now need to use `bbprep.generators.TorsionRange` and update its required terms (see the gist). 

This change was to avoid naming issues with `bbprep.TargetTorsion`

Let me know if this fixed your issue so I can close it.
